packages/base/matrix-event.gts

```diff
@@ -280,6 +255,7 @@ export interface CommandResultWithOutputContent {
     event_id: string;
   };
   data: {
+    commandRequestId: string;
```

Comment:
use our own custom rel_type `app.boxel.commandResultFor` with event_id and commandRequestId props.